### DISCUSSION TITLE
Adding a Discord server

### DISCUSSION THREAD

**Discussion by ScreamZ** [deepstreamIO/deepstreamIO.github.io#7] [Ideas]
Hi,

I'm pleased to see that deep stream is not dead. In fact, i really loved the way this framework works with unique features
- Distributed RPC and various things among other

Glad to see that more updates come thanks to @jaime-ez .

Could we add a Discord server to exchange about the product's future or provide real-time help ?

I'll be pleased to help contribute, especially on typescript typings for the front-end client. Or by making react / react native specific bindings ?

What do you think ?

**Comment by jaime-ez** [comment]
Hi @ScreamZ, thanks for your interest in the project!

I'm commited to keep maintaining deepstream (update deps, fix bugs) and help guide the discussion on new features, etc. It would be great if you could also be involved.

Regarding the discord server, I think it's best if we keep communication async and centered in github, using theses discussions or issues when required. That way we keep a smaller surface of interaction and can unify the knowledge base.

What do you mean by typescript typings for the front end? It already has, they come bunbled with it, take a look at the readme... Or you mean something else?

Also the client already works in react-native, at least last time I used a couple years ago... I'll take another look into it :)

The documentation regarding the react and react-native integration most certainly needs an update, if you can work on that it would be great

Best

**Comment by ScreamZ** [comment]
The Typescript typings should have more generic I guess, I'll take a look at this at some point :)
About react native we already talk here https://github.com/deepstreamIO/deepstream.io-client-js/issues/595

I'll check for the documentation when my instance works. Something more up to date with hooks supports for react.
